main-window.js

```diff
@@ -64,7 +64,7 @@ module.exports = function (config) {
   })
 
   var views = api.app.views(api.page.html.render, [
-    '/public', '/private', id, '/mentions'
+    '/polls', '/public', '/private', id, '/mentions'
```

Comment:
remove this before merging!

Reply:
actually having this entry be `process.env.START_PAGE` would be handy
I have something like this in Ticktack